.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "allo-isolate"]
+	path = allo-isolate
+	url = https://github.com/sunshine-protocol/allo-isolate
```

Comment:
This is temporary solution right? :) We should remove it when merging PR

Reply:
(Temporarily mark as unresolved because still see it in PR now, such that GitHub's PR protection can remind us of this. No worries now - just remove it before really merging PR)